src/test/java/com/petpool/application/util/ip/IpParserTest.java

```diff
@@ -5,32 +5,50 @@
 
 import java.net.InetSocketAddress;
 
-import static org.hamcrest.core.Is.is;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 
 public class IpParserTest {
     private IpParserImpl impl = new IpParserImpl();
+    private static String DEFAULT_PARSE_VALUE = "";
 
     @Test
     public void parserHeaderThenLocalHost() {
         HttpHeaders headers = new HttpHeaders();
         InetSocketAddress address = new InetSocketAddress("localhost", 5050);
         headers.setHost(address);
-        assertThat(impl.parse(headers), is("localhost"));
+        String ip = impl.parse(headers);
+        assertTrue(ip.equals("localhost"));
     }
 
     @Test
     public void parserHeaderThenHaveIp() {
         HttpHeaders headers = new HttpHeaders();
         InetSocketAddress address = new InetSocketAddress("10.0.0.1", 8080);
         headers.setHost(address);
-        assertThat(impl.parse(headers), is("10.0.0.1"));
+        String ip = impl.parse(headers);
+        assertTrue(ip.equals("10.0.0.1"));
     }
 
     @Test
     public void parserHeaderThenEmptyHeaders() {
         HttpHeaders headers = new HttpHeaders();
-        String parse = impl.parse(headers);
-        assertThat(parse, is(""));
+        String ip = impl.parse(headers);
+        assertTrue(ip.equals(DEFAULT_PARSE_VALUE));
```

Comment:
`assertTrue(ip.isEmpty)` or `assertEquals(actual, expected)` (first is preferable)
always write error messages for assertions: `assertTrue("The return values should be empty", ip.isEmpty())`